### ISSUE TITLE
State value has wrong type (js-controller 3.3.x)

### ISSUE THREAD

**Issue by thost96** [TA2k/ioBroker.nina#69]
js-controller: 3.3.4
Admin 5.0.11
Nina Adapter: 

Using new js-controller 3.3.4 will throw the following warnings:
```

nina.0 | 2021-05-01 12:21:29.450 | warn | (2310) State value to set for "nina.0.03252.identifierList" has wrong type "object" but has to be "string"
nina.0 | 2021-05-01 12:21:29.449 | warn | (2310) State value to set for "nina.0.03241.identifierList" has wrong type "object" but has to be "string"

```

Just for tracking see https://github.com/ioBroker/ioBroker.js-controller/issues/1299

**Comment by Feuer-sturm** [comment]
js-controller 3.3.11
Admin 5.1.7
Nina Adapter: 0.0.24

With current js-controller the message is now a little bit different: 

`State value to set for "nina.0.xxxx.identifierList" has to be stringified but received type "object"`

**Comment by TA2k** [comment]
fixed 0.0.25
